trello_to_mongo.py

```diff
@@ -45,3 +45,10 @@
 
 db.todo_items.insert_many( todos )
 
+# Add default statuses (update your own column names and substitute ObjectID for Trello idList value if importing data from Trello)
+db.todo_statuses.insertMany([
```

Comment:
Slight typo here, should be `insert_many` rather than `insertMany`, but I'm happy that with that tweak that let's me get to a working app